main.py

```diff
@@ -0,0 +1,123 @@
+import discord, datetime, time, random # Imports discord.py, datetime, time and random libraries
+from discord.ext import commands # Import the extension for discord.py - discord.ext command
+from discord import Member # import discord member, used in warning
+import json # import the json library
+import os
+
+from math import sqrt, cos, sin # import more math functions
+
+prefix = "m/" # Set the prefix. e.g "!sb "
+bot = commands.Bot(command_prefix=prefix) # Define what bot is
+bot.remove_command('help') # Remove the default help command from the Discord.py commands lib.
+botver = "1.0 [beta]" # Set the bot version number.
+
+start_time = time.time() # Starts the timer for the uptime of the bot.
+
+functions = ['+', '-', '*', '/', 'sqrt', 'cos', 'sin'] # math functions
+
+# create bot memory structure

```

Comment:
Is the warning system tested and working?

Reply:
yup

Reply:
Okay good :)